pkgs/top-level/all-packages.nix

```diff
@@ -281,6 +281,8 @@ in
   nixBufferBuilders = import ../build-support/emacs/buffer.nix { inherit (pkgs) lib writeText; inherit (emacsPackagesNg) inherit-local; };
 
   pathsFromGraph = ../build-support/kernel/paths-from-graph.pl;
+  
+  singularityTools = callPackage ../build-support/singularity { };
```

Comment:
`singularity-tools` is preferred spelling. (We're moving from camelCase to this-one for package names.)